samples/javascript/E1_SayHello/function.json

```diff
@@ -5,6 +5,5 @@
       "type": "activityTrigger",
       "direction": "in"
     }
-  ],
-  "disabled": false
```

Comment:
Does this mean we don't advertise `"disabled": false` in function.json anymore?  Not a problem, I'm just curious. :)

Reply:
@mhoeger oh wow, I did not know that! Thanks for the update. :)